src/NServiceBus.AcceptanceTests/Routing/When_broadcasting_a_command.cs

```diff
@@ -101,7 +103,7 @@ public Receiver2()
                 EndpointSetup<DefaultServer>(c =>
                 {
                     c.EndpointName("DistributingACommand.Receiver");
-                    c.ScaleOut().InstanceDiscriminator("2");
+                    c.EndpointInstanceId(_ => "2");
                 });
```

Comment:
Looks weird that for static IDs I have to use the func. Should we add a non-func overload?